sonus.js

```diff
@@ -38,7 +37,8 @@ for (let i = 0; i < config.speech.hotwords.length; i++) {
 
 const language = config.general.language
 const recordProgram = (os.arch() == 'arm') ? "arecord" : "rec"
-const sonus = Sonus.init({ hotwords, language, recordProgram }, speech)
+const device = (config.speech.device != "") ? config.speech.device : 'default'
+const sonus = Sonus.init({ hotwords, language, recordProgram, device }, speech)
```

Comment:
that's the part you changed... lol the rest is the cool shit... dynamic drop down for recording device in config UI